### PR TITLE
Image issue fixed

### PR DESCRIPTION
**Pull request by Abhishekg219**
Image issue fixed in this branch.

### CODE REVIEW

**Comment by victorleungtw** [comment]
It works and fix my issue. Thanks!

**Comment by Abhishekg219** [comment]
Thanks buddy :)

File: Pod/Classes/Banner.swift [hunk 1]
```diff
@@ -211,7 +212,7 @@ public class Banner: UIView {
     private func resetTintColor() {
         titleLabel.textColor = textColor
         detailLabel.textColor = textColor
-        imageView.image = shouldTintImage ? image?.imageWithRenderingMode(.AlwaysTemplate) : image
+        //        imageView.image = shouldTintImage ? image?.imageWithRenderingMode(.AlwaysTemplate) : image
```
Comment (harlanhaskins) [inline Pod/Classes/Banner.swift:215]: What happens if we want the image tinted?

**Comment by eastsss** [comment]
guys, just make `shouldTintImage = false` by default, that way people won't complain about image problem

**Comment by harlanhaskins** [comment]
Is this still an issue?

**Pull request closed without merging**